apps/sms/style/sms.css

```diff
@@ -232,6 +231,15 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   pointer-events: none;
 }
 
+[data-type="list"] .summary {
```

Comment:
I agree with @azasypkin here but let's do this in a separate bug as this one is 2.2+ and we should change all similar rules in one go.